app/apphandlers/embeddedAppHandler.go

```diff
@@ -62,6 +63,7 @@ func (a *embeddedApp) loadConfiguration() {
 	config.RegisterIntConfigVariable(0, &a.config.gatewayDSLimit, true, 1, "Gateway.jobsDB.dsLimit", "JobsDB.dsLimit")
 	config.RegisterIntConfigVariable(0, &a.config.routerDSLimit, true, 1, "Router.jobsDB.dsLimit", "JobsDB.dsLimit")
 	config.RegisterIntConfigVariable(0, &a.config.batchRouterDSLimit, true, 1, "BatchRouter.jobsDB.dsLimit", "JobsDB.dsLimit")
+	config.RegisterBoolConfigVariable(false, &a.config.forceStaticMode, false, "forceStaticModeProvider")
```

Comment:
**_Note:_** 
For testing purposes, we need to be able to support both a static ModeProvider and a Multi-tenant DeploymentType both at the same time, so that we don't need to do the _"etcd dance"_ when testing multi-workspace scenarios.

`processor_isolation_test.go` is such an example